### ISSUE TITLE
no-named-as-default-member

### ISSUE THREAD

**Issue by benmosher** [benmosher/eslint-plugin-import#205]
Given:
```js
// ./whatever.js
export { someThing }
export default class Whatever { /* ... */ }
```
Warn on:
```js
import Whatever from './whatever'
Whatever.someThing // Reported: someThing exported as named export alongside Whatever.
```

Would be unusable/noisy if there are static fields on a default export that match exported fields (i.e. lodash) but that seems fine.

I.e. 
```js
import _, { isEmpty } from 'lodash'
```
would report on `_.isEmpty`, etc. (assuming lodash exported ES modules). so you could only use the default export for chaining (i.e. `_(thing).isEmpty()`) which makes sense, otherwise you'd want `* as _` anyway.

**Comment by dmnd** [comment]
I have an implementation of this rule that I'll clean up and send as a PR either tonight or tomorrow.
